src/app/code/community/Zendesk/Zendesk/Block/Adminhtml/Config/Buttons/Sync.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright 2013 Zendesk.
```

Comment:
We've got a license fine in the root of the repo (https://github.com/zendesk/magento_extension/blob/master/LICENSE). I'm happy to not have these at the top of each file going forward.